src/main/interface/cli.c

```diff
@@ -1224,7 +1224,7 @@ static void cliRxRange(char *cmdline)
         ptr = cmdline;
         i = atoi(ptr);
         if (i >= 0 && i < NON_AUX_CHANNEL_COUNT) {
-            int rangeMin = 0, rangeMax = 0;
+            int rangeMin = PWM_PULSE_MIN, rangeMax = PWM_PULSE_MAX;
```

Comment:
Should this be in here?

Reply:
It snuck in as I was cleaning up a few other items. Happy to remove it, and submit an alternative PR (its relatively innocuous)